Changelog.md

```diff
@@ -1,3 +1,7 @@
+## Version 1.1.1
+### Bugfixes
+- Fix main view scrolling issue #42
```

Comment:
I think a bit more detail here would be nice. I suggest: `Overview on the left was not scrolled on smaller screens`

Reply:
Well it wasn't only the overview, everything else was scrolling incorrectly.